frontend/components/navbar/ProfilePic.tsx

```diff
@@ -133,47 +133,49 @@ export default function ProfilePic() {
               leaveTo="transform opacity-0 scale-95"
             >
               <MenuItems className="absolute right-0 mt-1 w-52 origin-top-right divide-y divide-gray-100 rounded-md bg-white shadow-lg ring-1 ring-black ring-opacity-5 focus:outline-none">
-                {loggedIn && (
-                  <div className="px-1 py-1">
-                    <Menu.Item>
-                      {({ active }) => (
-                        <Link
-                          href="/profile"
-                          className={`${
-                            active ? "text-white bg-highlight-yellow" : "text-black"
-                          } group flex w-full items-center rounded-md px-2 py-2 text-sm`}
-                        >
-                          <UserCircleIcon className="h-5 mr-2" />
-                          Profile
-                        </Link>
-                      )}
-                    </Menu.Item>
-                    <Menu.Item>
-                      {({ active }) => (
-                        <Link
-                          href="/settings"
-                          className={`${
-                            active ? "text-white bg-highlight-yellow" : "text-black"
-                          } group flex w-full items-center rounded-md px-2 py-2 text-sm`}
-                        >
-                          <Cog8ToothIcon className="h-5 mr-2" />
-                          Settings
-                        </Link>
-                      )}
-                    </Menu.Item>
-                  </div>
-                )}
+                <div className="px-1 py-1">
+                  {loggedIn && (
```

Comment:
This file is super duper hard to read with all the `loggedIn` and `!loggedIn` conditionals.. possibly we need 2 simple array of tuples, one for logged in and vice versa, e.g.
```
[
  ("Profile", "/profile", <UserCircleIcon/>),
  ("Create Event", "/create-event", <EditIcon/>)
  ...
]
```
such that we can map over the arrays to generate a dynamic drop down.

NOT IN SCOPE FOR THIS PR, just giving feedback

Reply:
that wasnt even me :(((((

Reply:
I COPIED EDWINNNNNNN

Reply:
make a TODO comment for this right above this piece of code and make a JIRA ticket linking to this direct PR comment.